ui/packages/catalog-ui-search/src/main/webapp/component/input/date/input-date.view.js

```diff
@@ -127,20 +138,47 @@ module.exports = InputView.extend({
     }
   },
   listenForChange: function() {
+    this.$el.on(
+      'dp.change',
+      function(e) {
+        if (e.oldDate === null) {
+          return
+        }
+
+        let datetimepicker = this.$el
+          .find('.input-group.date')
+          .data('DateTimePicker')
+
+        let newValue = this.getCurrentValue()
+
+        if (e.type === 'dp') {
```

Comment:
I don't think we need this type check anymore now that `listenForChange` only checks on `dp.change` and no longer on `click, keyup, etc`